build.gradle.kts

```diff
@@ -29,9 +29,7 @@ dependencies {
   implementation("javax.activation:activation:1.1.1")
   implementation("javax.transaction:javax.transaction-api:1.3")
   implementation("io.swagger:swagger-annotations:1.6.11")
-  implementation("org.springdoc:springdoc-openapi-ui:1.7.0")
-  implementation("org.springdoc:springdoc-openapi-kotlin:1.7.0")
-  implementation("org.springdoc:springdoc-openapi-data-rest:1.7.0")
+  implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.2.0")
```

Comment:
Was missed during Spring Boot 3 upgrade, should have been covered by running https://github.com/ministryofjustice/hmpps-tech-docs/blob/8e7c2f7d5c6591158586bc52a7ce83adeb0026ea/scripts/upgrade-spring-boot-3.bash#L93